EventsExpress.Db/Entities/ManageableEntity.cs

```diff
@@ -1,17 +1,19 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
+using EventsExpress.Db.EF;
 
 namespace EventsExpress.Db.Entities
 {
     public class ManageableEntity : BaseEntity
```

Comment:
We can remove this entity now. It is not required anymore, since we are doing the full tracking.